aiomisc/service/periodic.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from aiomisc import Service, PeriodicCallback
+
+
+log = logging.getLogger(__name__)
+
+
+class PeriodicService(Service):
+
+    __required__ = ('interval',)
+
+    interval = None  # type: float # in seconds
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.periodic = PeriodicCallback(self.callback)
+
+    async def start(self):
+        self.periodic.start(self.interval, loop=self.loop)
+        log.info('Periodic service %s started', self)
+
+    async def stop(self, err):
+        if self.periodic.task:
+            await self.periodic.task
```

Comment:
But `PeriodicCallback` by default does not wait - so there may be confusion.
Maybe add this to `PeriodicCallback.stop` if enabled by argument, say, `wait_task`?